dev-libs/pocl/files/pocl-3.1-nodebug.patch

```diff
@@ -0,0 +1,29 @@
+commit a13cb332d6678d4556d7319b284b77c371c4b91e
```

Comment:
Thanks. In future, could you include a link to the upstream commit at the top of the patch? I'll throw it in this time. Cheers!

Reply:
Sure, I'll keep that in mind, I directly pulled the commit from the repo after it got into master so that the commit ID is searchable in their upstream, but next time I'll add the link. Thanks for the review.